packages/FormElement/README.md

```diff
@@ -94,20 +95,16 @@ import Input from "@paprika/input";
 
 <FormElement label="Name">
   <FormElement.Content>
-    {({ idForLabel, ariaDescribedBy }) => (
-      <Input
-        id={idForLabel}
-        onChange={handleChange}
-        value={value}
-        placeholder="Form placeholder"
-        aria-describedby={ariaDescribedBy}
-        aria-required={hasRequiredLabel}
-        hasError={false}
-        isDisabled={isDisabled}
-        isReadOnly={isReadOnly}
-        size={size}
-      />
-    )}
+    <Input
```

Comment:
is this a breaking change or our FormElements still support the renderFunction?

Reply:
This will be a breaking change. I don't think we should encourage using the renderFunction as an acceptable api moving forward